aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -309,6 +314,7 @@ spec aptos_framework::account {
         );
 
         aborts_if account_scheme != ED25519_SCHEME && account_scheme != MULTI_ED25519_SCHEME;
+        ensures account_scheme == ED25519_SCHEME || account_scheme == MULTI_ED25519_SCHEME;
```

Comment:
Same as above, no need to add ensures because aborts_if condition is already defined.